src/test/java/SeleniumTechniques.java

```diff
@@ -1,21 +1,29 @@
 import org.openqa.selenium.By;
 import org.openqa.selenium.Keys;
 import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Action;
+import org.openqa.selenium.interactions.Actions;
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.openqa.selenium.support.ui.WebDriverWait;
 import org.testng.Assert;
 import org.testng.annotations.Test;
+import pages.HomePage;
+import pages.LoginPage;
 
 import java.time.Duration;
 import java.util.List;
 
 public class SeleniumTechniques extends BaseTest {
-
+    By playListLocator = By.cssSelector(".playlist:nth-child(3)");
+    By playlistInputLocator = By.cssSelector("input[name='name']");
+    By editPlayListLocator = By.xpath("//a[text()='Edited Playlist Name']");
+    By playlistSelector = By.cssSelector("section#playlistWrapper td.title");
+    By playBtnLocator = By.xpath("//span[@data-testid='play-btn']");
```

Comment:
POM pattern uses page objects that represent a web page (or a form within a page, if applicable) to separate test code (validations and test flow logic, for example) from page specific code. It does so by making all actions that can be performed on a page available as methods of the page object representing that page.

For example:



LoginPage = LoginTest

HomePage = HomeTest

CurrentQueuePage = CurrentQueueTest

![image](https://user-images.githubusercontent.com/104761482/215999561-2ab4c550-eaa8-4ec5-b065-c7c4fa66ec6e.png)